python/tvm/relay/build_module.py

```diff
@@ -707,24 +708,24 @@ def create_executor(kind="debug", mod=None, device=None, target="llvm", params=N
     -------
     executor : :py:class:`~tvm.relay.backend.interpreter.Executor`
     """
+    raw_targets = Target.canon_multi_target(target)
```

Comment:
so we require canon_multi_target, but we then immediate re-canonicalize. should we either assert it's canonicalized rather than force-converting it, or update docstring? perhaps here and in the executors, you really want a CanonicalizedTarget type?